ddtrace/contrib/grpc/patch.py

```diff
@@ -215,12 +217,18 @@ def _aio_client_channel_interceptor(wrapped, instance, args, kwargs):
 
     (host, port) = utils._parse_target_from_args(args, kwargs)
 
-    interceptors = create_aio_client_interceptors(pin, host, port)
+    dd_interceptors = create_aio_client_interceptors(pin, host, port)
+    interceptor_index = 3
```

Comment:
These hardcoded indexes can be trusted? Is there not a more durable way that we can rely on?